packages/cli/lib/projects.js

```diff
@@ -162,6 +162,26 @@ const validateProjectConfig = (projectConfig, projectDir) => {
   }
 };
 
+const verifyProjectExists = async (accountId, projectName) => {
```

Comment:
I created a new util here because `ensureProjectExists` wasn't able to satisfy my project verification use case

Reply:
I think you should be able to use `ensureProjectExists`. There's an arg `allowCreate: false` that forces the util to skip the "create project" flow when the project doesn't exist. 

You may just need to update the `ensureProjectExists` util to return a boolean, which IMO it should be doing anyways. If it's not too much of a lift, I'd prefer to only have one util for checking a project's existence.

Reply:
Definitely no lift there, wanted to make sure I didn't change any behaviors of existing utils but will get this addressed